assets/js/Containers/FrevaGPT/index.js

```diff
@@ -234,8 +244,15 @@ class FrevaGPT extends React.Component {
           <div className="d-flex justify-content-between">
             <h2 onClick={this.toggleBotSelect}>FrevaGPT</h2>
           </div>
+
+          {this.state.botOkay ? 
+            null : 
+            (<Alert key="botError" variant="danger">
+              The bot is currently not available. Please retry later.
+            </Alert>)
+          }
   
-          <Col md={4}>
+          <Col hidden={!this.state.botOkay} md={4}>
```

Comment:
Is the hidden render really necessary if the bot is unavailable? I would probably change the code here in a way, that we have two helper-render-functions where one is rendering just the Alert and the other one the rest and they will be called depending on `this.state.botOkay`.

Reply:
Seperated content into different render functions. Also covered displayed content during ping requests: https://github.com/FREVA-CLINT/freva-web/pull/61/commits/b2e35560a5f549453e98575847d0d1cbe44239d4

Reply:
Removed statement to hide content: https://github.com/FREVA-CLINT/freva-web/pull/61/commits/8c30e757d7b28174c327fde6f02c4c09e210fe49